app/views/staff/events/show.html.haml

```diff
@@ -103,24 +117,26 @@
               %tr
                 %td.text-primary
                   %strong CFP Closes Date:
-                - if event.closes_at.present?
+                - if event.closes_at && (event.closes_at > Time.current)
                   %td.set= link_to "Set", event_staff_edit_path
+                -elsif event.closes_at && (event.closes_at <= Time.current)
+                  %td.missing= link_to "Date has Passed", event_staff_edit_path
                 - else
                   %td.missing= link_to "Missing", event_staff_edit_path
               %tr
                 %td.text-primary
-                  %strong Session Formats:
-                - if event.session_formats.present?
-                  %td.set= link_to event.session_formats.count, '#'
+                  %strong Public Session Formats:
```

Comment:
Good point. Marty had mentioned using public_session_formats, but I do think it would be nice to show both. @mghaught How would you prefer this be structured?